### PR TITLE
📚 DOCS: add meeting minutes from 5th August Team Meeting

### PR DESCRIPTION
**Pull request by mmcky**
This PR adds minutes from today's monthly team meeting

### CODE REVIEW

**Comment by chrisjsewell** [comment]
Cheers!

**Comment by mmcky** [comment]
It looks like `:::` can't be used for specifying the `post` metadata for `ablog`

**Comment by choldgraf** [comment]
Hey all - thanks for posting these @mmcky - just to confirm, it looks like we did not get to discussion of anything in the agenda? It seems like most of the notes are in the "reports from team members" section. Is that right?

I think that is fine, though in the future I suggest that we get these meetings to a place where we can spend most of the time discussing things that require it, rather than just saying what we are up to! (That should be what the team activity board is for!)

**Comment by mmcky** [comment]
hey @choldgraf I will update the notes re: the `Agenda` section. We discussed the Activity Board and all agreed to use it this month. @chrisjsewell also had a great suggestion that we add it as a standing section to the monthly meeting.

**Comment by mmcky** [comment]
@choldgraf just added some additional notes. 

The reports section was a bit of a mix re: `reports` and discussion. We discussed the idea of `nested` code blocks for example and we reported that `ANU` is working on a proof of concept in `sphinx-exercise` for a gated approach to enabling code in `exercise` nodes for styling in `html` and `latex.

**Comment by chrisjsewell** [comment]
> It looks like `:::` can't be used for specifying the `post` metadata for `ablog`

#454 should fix this

File: docs/meetings/2021.md [hunk 1]
```diff
@@ -1,5 +1,110 @@
...
+        - Other options:
+            - Metadata? Adding tags, etc. (Similar to nbgrader notebooks)
+            - Glue? (currently restricted to python, cather than cellId or something)
+                - Similar ideas playing with at Curvenote! (copy/paste)
+        - Consideration is mostly from an authorship perspective
+
+- Steve Purves: Thebe
+    - got started and made some improvements to the build and docs -- yarn, CI issues, local kernel convenience commands for development
+    - PR for a small feature - progress indicator
+    - Collabrorating a bit with LibreText folks, would like to more and understand how they are using Thebe better
```
Comment (choldgraf) [inline docs/meetings/2021.md:52]:
```suggestion
    - Collaborating a bit with LibreText folks, would like to more and understand how they are using Thebe better
```

**Review by choldgraf — APPROVED** [review]
LGTM - thanks for posting these notes!

**Comment by choldgraf** [comment]
Thanks for filling this out - and sorry for the slowness. I'm working on transitioning 2i2c's fiscal sponsors and this is a *ton* of time that is sucking away all of my free time to work on EBP work stuff. Sorry about that!

**Comment by mmcky** [comment]
thanks @choldgraf

**Pull request merged**